packages/api-client/src/user/data/UserPropertiesSetData.ts

```diff
@@ -42,6 +54,7 @@ export interface WebappProperties {
       replace_inline: boolean;
     };
     interface: {
+      fontSize: FontSizePreference;
```

Comment:
Can be imported from the theme

Reply:
Not sure we should add a strict type here. It could just be a string. The consuming app is responsible for understanding the string it has put in there

Reply:
Sure, if the value will be defined and managed in the webapp, this can be a string type since the content has no relevance here.